tests/unit/Utils/DocLinkTest.php

```diff
@@ -11,9 +11,9 @@
  */
 class DocLinkTest extends \PHPUnit\Framework\TestCase {
 	public function testGetServerUrl() {
-		$expected = 'https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label';
+		$expected = 'https://doc.owncloud.com/server/10.12/admin_manual/installation/installation_wizard.html#strong-perms-label';
```

Comment:
```suggestion
		// This test constructs strings, from version and relative part and compares that
		// construct with the expected result. It does not check our actual servers. 
		$expected = 'https://doc.owncloud.com/server/10.12/admin_manual/installation/installation_wizard.html#strong-perms-label';
```
So we do not forget what this is about and version strings have no impact to reality.